### PR TITLE
Added debug.json to set of assets to help with server source tracing …

### PR DESCRIPTION
**Pull request by metzenseifner**
…when problems occur e.g. due to encoding

### CODE REVIEW

**Comment by metzenseifner** [comment]
@Arnei I resolved the conflicts, and this code should be good to go and compatible with the new configuration file.

**Comment by hex-m** [comment]
`createDebugJSONAtPath` is never called.

**Comment by metzenseifner** [comment]
Corrected. Thanks.

File: post-archive/post_archive.rb [hunk 1]
```diff
@@ -748,6 +767,21 @@ def cleanup(tmp_path, meeting_id)
...
 end
+# Add debugging information
+if ($config.dig(:addFiles, :sendDebugAttachment))
+  createDebugJSONAtPath(DEBUG_PATH, debugEntries)
+  if (File.file?(DEBUG_PATH))
+    BigBlueButton.logger.info("Adding debugging information as attachment asset to MP::" + mediapackage)
+    mediapackage = OcUtil::requestIngestAPI($oc_server, $oc_user, $oc_password, 
+                   :post, '/ingest/addAttachment', DEFAULT_REQUEST_TIMEOUT, 
+                   {:mediaPackage => mediapackage, 
+                   :flavor => "json/debug",
```
Comment (Arnei) [inline post-archive/post_archive.rb:778]: Might be worthwhile to make it more obvious that this attachment is specifically for BBB debuggin, e.g `json/bbb-debug`

**Review by Arnei — CHANGES_REQUESTED** [review]
My apologies for this way overdue review. Debug information to link Opencast events to the BBB servers whence they came seems to be very useful feature.

Unfortunately I can't merge this as is.

**Pull request closed without merging**